Source/UIScrollView+EmptyDataSet.m

```diff
@@ -738,7 +740,7 @@ - (void)didMoveToSuperview
     CGRect superviewBounds = self.superview.bounds;
     self.frame = CGRectMake(0.0, 0.0, CGRectGetWidth(superviewBounds), CGRectGetHeight(superviewBounds));
     
-    void(^fadeInBlock)(void) = ^{_contentView.alpha = 1.0;};
+    void(^fadeInBlock)(void) = ^{self->_contentView.alpha = 1.0;};
```

Comment:
you should use `__weak __typeof(_contentView) weakContentView = _contentView; instead`

Reply:
Thanks for the pointer!  I closed this and created a new PR with it updated